sdk/digitaltwins/azure-digitaltwins-core/src/test/resources/prerequisite/prerequisite readme.md

```diff
@@ -15,7 +15,7 @@
 
 ### Install Bicep
 
-- Install using the instructions in [bicep](https://github.com/Azure/bicep/blob/main/docs/installing.md)
+- Install using the instructions in [bicep](https://github.com/Azure/bicep)
```

Comment:
This page points to https://docs.microsoft.com/azure/azure-resource-manager/bicep/install for installing Bicep - in our case, we need to do a "manual install" to get the CLI commands directly. Suggest updating this to https://docs.microsoft.com/azure/azure-resource-manager/bicep/install#install-manually .